core/pages/Category.js

```diff
@@ -214,6 +222,26 @@ export default {
           EventBus.$emitFilter('category-after-load', { store: this.$store, route: this.$route })
         }
       })
+    },
+    refreshData () {
```

Comment:
Please rename this one to `onUserPricesRefreshed` + add the comment (why we're reloading the products in that case) to make it easier to understand